README.md

```diff
@@ -11,14 +11,20 @@
   <br />
   <br />
   <a href="https://circleci.com/gh/FormidableLabs/urql">
-    <img alt="Test Status" src="https://circleci.com/gh/FormidableLabs/urql.svg?style=shield" />
-  </a> <a href="https://github.com/FormidableLabs/urql#maintenance-status">
-    <img alt="Maintenance Status" src="https://img.shields.io/badge/maintenance-active-green.svg" />
```

Comment:
This badge should be retained on every Formidable repo :)

Reply:
Is that a thing we do? 

I don't see what value it provides for actively maintained projects - a [deprecation notice](https://github.com/andyrichardson/atom-updater-linux) when a project is no longer maintained however 👌

Reply:
Yep, it's been added to all our packages; here's one of the PRs: https://github.com/FormidableLabs/urql/pull/226
Those correspond to our "OSS Maintenance Levels" to transparently communicate which projects are also stable or experimental. Deprecated (or "Archived" in our case) is just one of four statuses.